### PR TITLE
Minor doc changes.

### PR DESCRIPTION
**Pull request by rocky**
* Preface all environement variable that need to be set with MYTHRIL
* Remove mention of email and API in README and example programs
* We don't do escaped source in JSON
* Mythril Platform -> MythOS

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ Input parameter handling is minimal and command options are lacking so
 we can focus on how to set up and call the client.
 
 
-* analysis: Submit a JSON with Solidity source and EVM bytecode information to the Mythril Platform and retrieve results
-* api-version: Retrieve Mythril Platform API version information. JSON is output.
-* openapi-spec: Retrieve the current Mythril Platform openapi specification. YAML is output
+* [analysis](https://github.com/ConsenSys/armlet/blob/jwt-auth/example/analysis): Submit a JSON with Solidity source and EVM bytecode information to the MythOS and retrieve results
```
Comment (fgimenez) [inline example/README.md]: All the links should point to `master`, not to the `jwt-auth` branch https://github.com/ConsenSys/armlet/blob/master/example/analysis

File: example/analysis [hunk 2]
```diff
@@ -6,9 +6,9 @@ const fs = require('fs')
 function usage() {
     console.log(`usage: ${process.argv[1]} [*mythril-api-json-path*] [*timeout-secs*]
 
-Run  Mythril Platform analyses on *mythril-api-json-path*
+Run  MythOS analyses on *mythril-api-json-path*
 
-Set environment variables EMAIL and MYTHRIL_API_KEY before using.
+Set environment variables MYTHRIL_ETH_ADDRESS and MYTHRIL_PASSWORD before using.
```
Comment (fgimenez) [inline example/analysis]: Should we mention at some point that `email` is also accepted instead of `ethAddress`?

**Review by fgimenez — COMMENTED** [review]
Looks good thanks! Just fixing the links should be enough.

File: example/README.md (hunk 1 above)
Comment (rocky) [reply example/README.md]: Good catch. Fixed now.

**Review by fgimenez — APPROVED** [review]
Nice, thanks!

**Pull request merged**